packages/streamable/streamable.v0.16.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Jane Street developers"
+authors: ["Jane Street Group, LLC"]
+homepage: "https://github.com/janestreet/streamable"
+bug-reports: "https://github.com/janestreet/streamable/issues"
+dev-repo: "git+https://github.com/janestreet/streamable.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/streamable/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"            {>= "4.14.0"}
+  "async_kernel"     {>= "v0.16" & < "v0.17"}
+  "async_rpc_kernel" {>= "v0.16" & < "v0.17"}
+  "base"             {>= "v0.16" & < "v0.17"}
+  "core"             {>= "v0.16" & < "v0.17"}
+  "core_kernel"      {>= "v0.16" & < "v0.17"}
+  "ppx_jane"         {>= "v0.16" & < "v0.17"}
+  "dune"             {>= "2.0.0"}
+  "ppxlib"           {>= "0.28.0"}
+]
+available: arch != "arm32" & arch != "x86_32"
+synopsis: "A collection of types suitable for incremental serialization."
```

Comment:
```suggestion
synopsis: "A collection of types suitable for incremental serialization"
```

Synopsis shouldn't end with a punctuation (or else rendering for e.g. web page entries can get messed up)